app/src/main/java/com/toast/android/gamebase/sample/ui/login/LoginScreen.kt

```diff
@@ -168,7 +168,7 @@ fun OutlineLoginButton(
             Spacer(Modifier.width(dimensionResource(id = R.dimen.login_screen_icon_text_space)))
             Text(
                 color = Color.Black,
-                text = stringResource(id = R.string.sign_in_button, idp.capitalize()),
+                text = stringResource(id = R.string.sign_in_button, idp.replaceFirstChar { firstChar -> firstChar.uppercase() }),
```

Comment:
[MAY] 함수 이름이 이미 하고자 하는 바를 자세히 알려주고 있으므로, 굳이 람다 변수를 override 선언할 것 없이 it 을 그대로 쓰는 쪽이 간결해지지 않을까 생각합니다.
```suggestion
                text = stringResource(id = R.string.sign_in_button, idp.replaceFirstChar { it.uppercase() }),
```